specification/webdriver-classic.bs

```diff
@@ -0,0 +1,177 @@
+<pre class="metadata">
+Title: webdriver-classic
+Shortname: wecg-webdriver-classic
+Level: 1
+Group: wecg
+Status: w3c/CG-DRAFT
+URL: https://w3c.github.io/webextensions/specification/webdriver-classic.html
+Editor: Kiara Rose, Apple, kiara_rose@apple.com
+Abstract: This specification defines how web extensions should be loaded and unloaded in WebDriver Classic.
+Repository: w3c/webextensions
+</pre>
+
+  <section>
+    <h3> Endpoints</h3>
+
+    <table class=simple>
+     <tr>
+      <td>POST</td>
+      <td>/session/{<var>session id</var>}/webextension</td>
+      <td>Load WebExtension</td>
+     </tr>
+
+     <tr>
+      <td>DELETE</td>
+      <td>/session/{<var>session id</var>}/webextension/{<var>extension id</var>}</td>
+      <td>Unload WebExtension</td>
+     </tr>
+    </table>
+  </section>
+
+  <section>
+    <h3>Errors</h3>
+    <table class=simple>
+     <tr>
+      <th>Error Code
+      <th>HTTP Status
+      <th>JSON Error Code
+      <th>Description
+     </tr>
+
+     <tr>
+      <td>no such web extension
+      <td>404
```

Comment:
Could we mention what error from the BiDi spec each of these refers to? It actually looks like if you click the errors in the BiDi spec, it links you to a page with an existing mapping to HTTP status codes: https://www.w3.org/TR/webdriver-bidi/#expand-a-web-extension-data-spec